### ISSUE TITLE
BlendingEditor is not rendered by accessing it trough Launchpad

### ISSUE THREAD

**Issue by PatrykSzwer** [Starcounter/Blending#184]
Related to https://github.com/Starcounter/Blending/pull/183#pullrequestreview-116863321

**Comment by kegor** [comment]
I see that all works as expected with disabled viewKeeper:
![switching_apps_using_launchpad_disabled_viewkeeper](https://user-images.githubusercontent.com/17431807/39572671-c80dbf5e-4ed8-11e8-9bfb-cc2161984b60.gif)

**Comment by PatrykSzwer** [comment]
If that's the case please investigate further, since It's not expected behavior.

**Comment by kegor** [comment]
This is what I found at Monday:
1. Transitions between pages of BE:
1.1. Always empty `keptView.OriginalResponse` until following check: https://github.com/Starcounter/Blending/blob/c5a1eb993c21e910d5fa6b0b079fa1fa516d2b78/src/BlendingProvider/Website/ViewModels/ViewKeeper.json.cs#L64
1.2. This is not noticeable in this case (transitions inside the BE) because each transition essentially changes the `keptView.ResponseJsonString` and therefore we go to the block in which the` keptView.OriginalResponse` is assigned, and it turns out that the ViewKeeper is not particularly in demand in this case.
2. Transitions between the apps using Launchpad:
1.2. The same with `keptView.OriginalResponse` it is empty, and judging by the debug it is empty already here: https://github.com/Starcounter/Blending/blob/c5a1eb993c21e910d5fa6b0b079fa1fa516d2b78/src/BlendingProvider/Website/Api/ContentHandlers.cs#L284, after clicking on the application icon to return back.
2.2. Here it is certainly noticeable, because instead of the normal `keptView.OriginalResponse`, we see: `{}`
3. Test method `SwitchWrappingLayouts_PreservesViewModelState`
3.1. After the first filling (the first approach to `/Form`) `keptView.OriginalResponse` is stably filled.
3.2. I also noticed that during the test debug, I went every time I clicked on the links to the line https://github.com/Starcounter/Blending/blob/c5a1eb993c21e910d5fa6b0b079fa1fa516d2b78/src/BlendingProvider/Website/Api/ContentHandlers.cs#L265, and in cases above it goes there once at the first page load.
4. I noticed that on https://github.com/Starcounter/Blending/blob/c5a1eb993c21e910d5fa6b0b079fa1fa516d2b78/src/BlendingProvider/Website/Api/ContentHandlers.cs#L284 it turns out to be empty that `viewKeeper.KeptViews.OriginalResponse` which refers to the URL to which we navigate i.e. for example, for a 2nd case with a Launchpad, if we switch from `/SignIn` to `/BlendingEditor`, then for OriginalResponse, there will already be {} and for `/SignIn` it will still be full, and will stop when you go back to `/SignIn`.
